example/src/components/Buttons/index.js

```diff
@@ -1,53 +1,40 @@
-              fill="black"
-            />
-          </svg>
-        </Span>
+        <Span>Download Passport</Span>
+        <Span>↓</Span>
       </Button>
-    </Box>
 
-    <Box mb="4">
-      <Button onClick={() => console.log("Downloading...")} disabled>
-        <Span>Go</Span>
-        <Span>→</Span>
-      </Button>
-    </Box>
-    <Box mb="4">
-      <Button onClick={() => console.log("Downloading...")} error>
-        <Span>Error</Span>
-        <span role="img" aria-label="Error">
-          ⚠️
-        </span>
-      </Button>
+      <Box mb="4">
+        <Button onClick={() => console.log("Downloading...")} isLoading={true}>
```

Comment:
I think on Loading, it should have a cursor: progress

Reply:
Updated, should be good to merge.